web/src/features/moreCast2/components/StationPanel.tsx

```diff
@@ -86,8 +87,19 @@ const StationPanel = (props: StationPanelProps) => {
       newSelectedStations.push(station)
     }
     dispatch(selectedStationsChanged(newSelectedStations))
+    setSelectAll(newSelectedStations.length > 0)
```

Comment:
Ya it could be, I was just trying to think of different scenarios. The one I was thinking of:
Forecaster has a big group of stations turned on (but not all of them), then wants to narrow it down to 1 or 2. They can turn them all off then turn on the 1 or 2 they want. 
vs
Turning them all on first, then clearing selection, or turning them off individually.

There's also an argument for the opposite functionality, having that button either turn all of them off or all of them on. 

I'm completely open to either

Reply:
Though I guess my initial thought does go against the acceptance criteria, I just realized

Reply:
I was more thinking that this will `setSelectAll` to `true` if `newSelectedStations.length` is 1, even if that is the only selected station?

In other words where do we capture the condition:
- `setSelectAll` to `true` iff `selectedStations.length` == `allStations.length`

Reply:
True on the first statement for sure

Maybe dev chat when you have a few mins? 

Not sure if you saw my comment edit, but this is the question really: It just depends what we want the behaviour to be when only a subset are turned on. Turn the rest on, or clear the selection

Reply:
> It just depends what we want the behaviour to be when only a subset are turned on. Turn the rest on, or clear the selection

I see now, I think turn the rest on.